draft-ietf-quic-recovery.md

```diff
@@ -320,10 +320,17 @@ SHOULD be set for twice the newly computed smoothed RTT.
 
 When crypto packets are sent, the sender SHOULD set a timer for the crypto
 timeout period.  Upon timeout, the sender MUST retransmit all unacknowledged
-CRYPTO data by calling RetransmitAllUnackedCryptoData(). On each consecutive
-expiration of the crypto timer without receiving an acknowledgement for a new
-packet, the sender SHOULD double the crypto handshake timeout and set a timer
-for this period.
+CRYPTO data if possible.
+
+Until the server has confirmed the path, it must not send more than 3
+times the number of received bytes, as specified in the {{QUIC-TRANSPORT}}.
+If not all unacknowledged CRYPTO data can be sent, then all CRYPTO data in
+Initial encryption should be retransmitted.  If no bytes may be sent,
+then no alarm should be armed until bytes have been received by the peer.
+
+On each consecutive expiration of the crypto timer without receiving an
+acknowledgement for a new packet, the sender SHOULD double the crypto
+handshake timeout and set a timer for this period.
```

Comment:
The timeouts are still inconsistently named.  We have "crypto handshake timeout" here, but "crypto timeout" on 321/322.  I think that "crypto retransmission timeout " is most correct.  You should fix the section title as well.